src/pages/evenements/index.tsx

```diff
@@ -22,11 +93,82 @@ export default function PageEvenements() {
               buttonLabelSecondary='Je trouve un événement ma Mission Locale'
               buttonHref='https://mesevenementsemploi.pole-emploi.fr/mes-evenements-emploi/evenements'
               buttonHrefSecondary='https://40-ans.unml.info/le-programme'
-              imgSrc='/images/évènements.webp' />
+              imgSrc='/images/évènements.webp'/>
           </main>
         </>
       }
-      { displayRechercheEvenement && <h1>Page recherche évenement en Construction</h1> }
+      {displayRechercheEvenement && <>
+        <HeadTag
+          title={'Rechercher un évènement | 1jeune1solution'}
+          description="Des centaines d'évènements de recrutement pour tous les jeunes, partout en France"/>
+        <main id="contenu">
+          <LightHero
+            primaryText="Des centaines d'évènements de recrutement"
+            secondaryText="pour tous les jeunes, partout en France"
+          />
+          <InstantSearch searchClient={searchClient} indexName={MEILISEARCH_INDEX}
+            routing={MEILISEARCH_QUERYPARAMS_ROUTING_ENABLED}>
+            <Configure hitsPerPage={HITS_PER_PAGE}/>
+            <div className={'separator'}>
+              <Container className={styles.evenementFormWrapper}>
+                <form className={styles.evenementForm}>
+                  <div className={styles.formWrapper}>
+                    <div className={styles.form}>
+                      <label>Mot-clé, métier, accompagnement…</label>
+                      <SearchBox
```

Comment:
on a un gros soucis avec SearchBox car c'est wrappé dans un form et on en a déjà un, du coup on a un form dans un form...je ne sais pas s'il y a de solutions....

Reply:
non je ne peux pas modifier la searchbox à moins de la reconstruire totalement à la main :/ 
ça serait un nice to have